obspy/io/focmec/core.py

```diff
@@ -0,0 +1,384 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+FOCMEC file format support for ObsPy
+
+:copyright:
+    The ObsPy Development Team (devs@obspy.org)
+:license:
+    GNU Lesser General Public License, Version 3
+    (https://www.gnu.org/copyleft/lesser.html)
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA @UnusedWildImport
+
+import re
+import warnings
+
+import numpy as np
+
+from obspy import UTCDateTime, Catalog, __version__
+from obspy.core.event import (
+    Event, FocalMechanism, NodalPlanes, NodalPlane, Comment, CreationInfo)
+
+
+# XXX some current PR was doing similar, should be merged to
```

Comment:
what about this ? is this still valid?

Reply:
Yeah I think so, just a note for future reference I guess